### PR TITLE
YETUS-1247. Update dependencies for 2024-10

### PR DESCRIPTION
**Pull request by aw-was-here**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -250,7 +250,7 @@
         <artifactId>apache-rat-plugin</artifactId>
         <version>${maven-rat-plugin.version}</version>
         <configuration>
-          <excludeSubprojects>false</excludeSubprojects>
+          <!-- <excludeSubprojects>false</excludeSubprojects> -->
```
Comment (ndimiduk) [inline pom.xml:253]: Just delete it?

File: precommit/src/main/shell/plugins.d/codespell.sh [hunk 2]
```diff
@@ -83,7 +83,7 @@ function codespell_logic
   "${CODESPELL}" \
     --disable-colors \
     --interactive 0 \
-    --quiet-level 2 \
+    --quiet-level 34 \
```
Comment (ndimiduk) [inline precommit/src/main/shell/plugins.d/codespell.sh:86]: Wow that's an esoteric means of configuration.

**Review by ndimiduk — APPROVED** [review]
Up and to the right!

File: pom.xml (hunk 1 above)
Comment (aw-was-here) [reply pom.xml:253]: Doh. I forgot about this one. yes, I'll do a quick update to this PR to just delete it.  Thanks!

**Pull request merged**